lib/sinon-mongo.js

```diff
@@ -50,9 +50,11 @@ const install = sinon => {
     if (!result) result = [];
     if (result.constructor !== Array) result = [result];  
 
-    return {
+    const cursor = {
```

Comment:
Thanks for the PR!

I think we should rather setup sort to be a stub that returns itself, as the code under test would typically look like:
```
collection
  .find({name: 'foo'}, {email: 1, name: 1})
  .sort({email:1})
  .toArray();
```

So we can create the stub as
```
const cursor = {
  sort: sinon.stub().returnsThis(),
  toArray: () => Promise.resolve(result)
};
return cursor;
```

That way we can easily test code that uses sort at the same time that we can assert sort was called as expected